src/server/game/Server/Packets/MiscPackets.cpp

```diff
@@ -201,3 +201,41 @@ void WorldPackets::Misc::WorldTeleport::Read()
     _worldPacket >> Pos;
     _worldPacket >> Facing;
 }
+
+WorldPacket const* WorldPackets::Misc::CorpseReclaimDelay::Write()
+{
+    _worldPacket << Remaining;
+
+    return &_worldPacket;
+}
+
+WorldPacket const* WorldPackets::Misc::DeathReleaseLoc::Write()
+{
+    _worldPacket << MapID;
+    _worldPacket << Loc;
+
+    return &_worldPacket;
+}
+
+WorldPacket const* WorldPackets::Misc::PreRessurect::Write()
+{
+    PlayerGUID.WriteAsPacked();
```

Comment:
What Shauren meant was actually:
```c++
_worldPacket << PlayerGUID.WriteAsPacked();
```
Otherwise nothing gets actually written to the ByteBuffer.